sdk/servicebus/service-bus/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 7.0.0-preview.2 (Unreleased)
 
+- Any error caused by a connection disconnecting is retried as per the user-provided retry options and the failures would get surfaced to the user's registered error handler
```

Comment:
This makes a user feel that this was not being done before...

Also, since the change we are doing is not anything different from what is in Track 1, I wonder if we should even mention this...

While it is generally true that changelog for a version should reflect the diff between it and the previous version, in case of previews I believe we should focus on what's different between this and the stable version

Thoughts...?

Reply:
Okay, makes sense, I agree with removing.

Reply:
Closing this PR in favor of the unified PR #8580.
Please continue over there.